trismart_smartboard_integration/models/res_partner.py

```diff
@@ -0,0 +1,10 @@
+from odoo import api, fields, models, tools, SUPERUSER_ID
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    sb_lead_id = fields.Char('Smartboard Lead ID', readonly=True)
+    sync_status = fields.Selection([
+        ('1', 'Needs Update'), ('2', 'Updated')])
```

Comment:
When will the sync_status of partner change to Updated?

Reply:
Our goal is to create a template project with empty content, the status will be "Need Update". In the second part, when user pass the details of the project, we fetch it and change its status to "Updated"